fleet-charts/argocd-clusterprofile-syncer/Chart.yaml

```
@@ -14,7 +14,7 @@
 
 apiVersion: v2
 name: argocd-clusterprofile-syncer
-version: 0.0.1
+version: 0.0.2
```

why not keeping them in sync?
```suggestion
version: 0.1.0
```

---

We could, but keeping them in lock-step could get complicated if/when we want to update the chart, but there is no update to the application/image.  The `appVersion` field is specifically to indicate the application's version, as opposed to the chart's version.

---

Does it have to follow major.minor.patch? One option is to add suffix like 0.1.0-mco.1 (probably not worth inventing a new version scheme though).

---

Yeah, looks like that field requires SemVer 2, so you example should be accepted.  At a very least we can always add a "build" if needed: `0.1.0+1`